tests/container_t.cpp

```diff
@@ -130,10 +130,11 @@ CASE("containers/linear/simple") {
   auto l2 = model->add(Linear(3, 5).make(), "l2");
   auto l3 = model->add(Linear(5, 100).make(), "l3");
 
-  auto x = Var(at::CPU(at::kFloat).randn({1000, 10}));
-  x = l1->forward({x})[0].clamp_min(0);
-  x = l2->forward({x})[0].clamp_min(0);
-  x = l3->forward({x})[0].clamp_min(0);
+  Variable x = Var(at::CPU(at::kFloat).randn({1000, 10}))
+    .m(l1->functor()).m(at::relu)
+    .m(l2->functor()).m(at::relu)
+    .m(l3->functor()).m(at::relu)
```

Comment:
You could provide a custom `m` in `Variant` though. Not sure whether that would make things easier or more complicated...

Reply:
As in, randn returns a tensor, so I can't dot operator on it, or specialize it for Containers since it's in PyTorch.. Variant already provides the same .m operator.